src/libs/seedData.ts

```diff
@@ -0,0 +1,35 @@
+import UserRepository from '../repositories/user/UserRepository';
+
+const userRepository = new UserRepository();
+
+const seedData = async () => {
+    try {
+        const res = await userRepository.count();
+        console.log('resolve', typeof res);
+        console.log('Total number of documents: ', res);
+        if (res === 0) {
+            console.log('Data seed in progress...');
+            userRepository.create(
```

Comment:
Create a array for seed data and iterate over array and call userRepository.create for each element.

Reply:
Done with changes.